providers/logging/winstonConfig.js

```diff
@@ -11,7 +11,7 @@ function factory(options) {
   const realOptions = options || {
     key: config.get('APPINSIGHTS_INSTRUMENTATIONKEY'),
     echo: false,
-    level: 'info'
+    level: config.get('APPINSIGHTS_EXPORT_LOG_LEVEL') || 'info'
```

Comment:
Nice addition to be able to set the log level from the configs.  Quite handy.  ❤️

Reply:
Thanks, I will add to the infrastructure docs